### ISSUE TITLE
Link this video into the resources page

### ISSUE THREAD

**Issue by mpdehaan** [ansible/ansible#2055]
http://www.youtube.com/watch?v=up3ofvQNm8c&feature=youtu.be&a

ansible.cc and online docs, better than I can do, I think :+1:

**Comment by mpdehaan** [comment]
and this.

http://www.youtube.com/watch?v=J255Sud4rDI …

**Comment by leucos** [comment]
It would be nice if @skvidal (?) could provide the slides for the FUDcon video. It's pretty hard to follow like that.

**Comment by mpdehaan** [comment]
I have the google slides link from Seth but I'm not going to repost incase
I misunderstood the link permissions (they seem to be set correctly)

I'll let him post.

**Comment by skvidal** [comment]
https://docs.google.com/presentation/d/1rER6QB3f4xpS5ymOd0bN-bTJoj4Rx90qFCZ9knRxZvI/edit?usp=sharing

**Comment by leucos** [comment]
Great ! Thanks @skvidal :+1:

**Comment by mpdehaan** [comment]
done
